envi/archs/thumb16/disasm.py

```diff
@@ -926,6 +931,8 @@ def adv_simd_32(va, val1, val2):
     ('11110101011',         (85,'sbc',      dp_mod_imm_32,        IF_THUMB32)),
     ('11110101101',         (85,'sub',      dp_mod_imm_32,        IF_THUMB32)),  # cmp if rd=1111 and s=1
     ('11110101110',         (85,'rsb',      dp_mod_imm_32,        IF_THUMB32)),
+    ('11100',       (INS_B,  'b',       pc_imm11,           envi.IF_NOFALL)),        # B <imm11>
```

Comment:
bugfix: need these in here too, unless we merge thumb1 and thumb2, which may be possible at a later date.